superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/transformProps.ts

```diff
@@ -290,12 +291,24 @@ export default function transformProps(
   const showValueIndexesB = extractShowValueIndexes(rawSeriesB, {
     stack,
   });
+
+  const metricMapFunc = (metric: string | AdhocMetric) => {
+    if (metric.hasOwnProperty('label')) {
+      return (metric as AdhocMetric).label;
```

Comment:
I think we already have a util `getMetricLabel` for that

Reply:
Ah, yes. How should that function work with the verboseMap, though?